src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,27 @@
+// Libraries
 import React from 'react';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+// Styles
+import './CommentList.scss';
+
+// Types
+import { Comment } from '../../types/Comment';
+
+// Components
+import { CommentInfo } from '../CommentInfo';
+
+type Props = {
+  comments: Comment[];
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <ul className="CommentList">
+    {comments.map((comment: Comment) => (
+      <CommentInfo
```

Comment:
Add a key prop to each CommentInfo component with a unique value, like the comment id.